### DISCUSSION TITLE
Stop opening tickets in new tab rather show the panel from the right.

### DISCUSSION THREAD

**Discussion by Dnathan33** [community/community#16201] [Projects and Issues]
Stop opening tickets in new tab rather show the panel from the right.

**Comment by MrTJP** [comment]
Seems to only happen for Board views for me. And when the tab opens, there's still no view of the ticket. Just back to the original board view.

**Comment by egost** [comment]
Duplicate here: https://github.com/github/feedback/discussions/16164
